docs/index.asciidoc

```diff
@@ -26,6 +26,26 @@ Generate heartbeat messages.
 The general intention of this is to test the performance and
 availability of Logstash.
 
+[id="plugins-{type}s-{plugin}-ecs"]
+==== Elastic Common Schema (ECS)
+
+This plugin could provide a field, originally named `clock`, to track `epoch` or `sequence` incremental numbers. When
+<<plugins-{type}s-{plugin}-ecs_compatibility,ECS compatibility mode>> is enabled that value is now present in the
+event's `[event][sequence]` subfield.
+
+When <<plugins-{type}s-{plugin}-ecs_compatibility,ECS compatibility mode>> is enabled the use of `message` as
+selector of sequence type is not available and only <<plugins-{type}s-{plugin}-sequence>> is considered. In this
+case if `message` contains sequence selector strings it's ignored.
+
+The existing `host` field is moved to `[host][name]` when ECS is enabled.
+
+.Metadata Location by `ecs_compatibility` value
+[cols="<l,<l,e,<e"]
+|============================================================================================================
+|`disabled`    |`v1`              |Availability                       |Description
```

Comment:
```suggestion
[cols="<l,<l,e,<e"]
|============================================================================================================
|`disabled`    |`v1`              |Availability                       |Description

```

Reply:
Removes table title. Adds return after table headings to make them format as bold.